### DISCUSSION TITLE
make `--layout-content-width` adjustable

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [forem/forem#20573] [Feature Requests]
It is always horrendous to what to have so much wasted space in articles. While on the other hand, having to deal with scrolling horizontally in source code. it would be nice if there was an option to release more of the space available by having a possibility either to influce the `--layout-content-width` or some general layout parameters. The 3 col layout with the excessive margins caused by margin auto is really difficult.

![grafik](https://github.com/forem/forem/assets/86342787/bd174452-538e-4ea8-b4a6-ed29dec4c8a9)

**Comment by sschneider-ihre-pvs** [comment]
Disabling the complete layout is more readable

![grafik](https://github.com/forem/forem/assets/86342787/01028d12-9878-48e5-bad0-43ab98d27a2c)

**Comment by pBouillon** [comment]
Please make it happens 🙏

**Comment by tilo** [comment]
It is 2026. Screens are big and wide. Fixed-width layout is so 2000's -- an ancient relic!!
Adjustable content width is not difficult! You can do better than Medium.
